app/views/layouts/application.html.haml

```diff
@@ -39,4 +39,10 @@
         %span= msg
     = yield
 
+- content_for :footer_top do
+  %p
+    Problems using this service? Email
+    =mail_to 'isobel.croot@digital.education.gov.uk'
```

Comment:
Can we extract this to an environment variable and reuse so it's not stored in the codebase.

Reply:
Agreed.